.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -1,66 +1,67 @@
-name: "\U0001F41E Bug Report"
-description: Create a report to help us improve lemmy-ui
-title: "[Bug]: "
+name: "Bug report"
+description: Create a bug report to help us improve Lemmy-UI!
+title: ""
 labels: ["bug", "triage"]
 body:
   - type: markdown
     attributes:
       value: |
-        Found a bug? Please fill out the sections below. 👍
-        Thanks for taking the time to fill out this bug report!
-        For backend issues, use [lemmy](https://github.com/LemmyNet/lemmy/issues/new/choose)
+        Thanks for taking the time to help improve Lemmy-UI by reporting a bug!
```

Comment:
Making the backend details one of the steps below, so no need to have additional text here.